gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-kafka/src/main/java/io/gravitee/plugin/endpoint/kafka/KafkaEndpointConnector.java

```diff
@@ -69,6 +70,7 @@ public class KafkaEndpointConnector extends EndpointAsyncConnector {
     static final String CONTEXT_ATTRIBUTE_KAFKA_GROUP_ID = KAFKA_CONTEXT_ATTRIBUTE + "groupId";
     static final String CONTEXT_ATTRIBUTE_KAFKA_RECORD_KEY = KAFKA_CONTEXT_ATTRIBUTE + "recordKey";
     private static final String ENDPOINT_ID = "kafka";
+    private static final String KAFKA_CLIENT_IDENTIFIER = UUID.randomUUID().toString();
```

Comment:
I don't think it should be static but an instance attribute instead.
Each instance of connector should be identified as a specific kafka client imho.

Reply:
Yes agreed.